server/entrypoint.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
Tohle ten skript rozbije, protože se v něm používá bash syntax.

` ./entrypoint.sh: 23: ./entrypoint.sh: [[: not found `